uiowa/uiowa
# Some November dependency updates not handled by dependabot

Updates the `diff` and `lb_direct_add` modules.

# How to test
- Review dependency updates. I tested `lb_direct_add` update already.
- `ddev composer install`
- `ddev blt ds`
- `ddev drush @default.local uli node/1/revisions`
- Select 2 revisions and compare them. Confirm no errors and display looks as expected.
